src/logger.ts

```diff
@@ -0,0 +1,7 @@
+export const log = {
+  debug: (message: string): void => {
+    if (process.env.ADMIN_BRO_EXPRESS_DEBUG) {
```

Comment:
I think it would be awesome to document this env in readme or somewhere :wink:

Reply:
done